google/resource_sql_database.go

```diff
@@ -86,8 +91,9 @@ func resourceSqlDatabaseRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	database_name := d.Get("name").(string)
-	instance_name := d.Get("instance").(string)
+	s := strings.Split(d.Id(), ":")
+	instance_name := s[0]
```

Comment:
We should validate that there are exactly 2 elements in this array, and return an error specifying the format otherwise. That way, users that discover they can import the resource through [the Importability page](https://www.terraform.io/docs/import/importability.html) and try just a `name` will find that they also need to specify `instance`.